src/serac/physics/solid_mechanics.hpp

```diff
@@ -599,9 +598,11 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
   {
     residual_->AddBoundaryIntegral(
         Dimension<dim - 1>{}, DependsOn<0, 1, 2, active_parameters + NUM_STATE_VARS...>{},
-        [this, traction_function](auto x, auto n, auto, auto, auto shape, auto... params) {
-          auto p = get<VALUE>(shape);
-          return -1.0 * traction_function(x + p, n, ode_time_point_, params...);
+        [this, traction_function](auto X, auto /* displacement */, auto /* acceleration */, auto shape, auto... params) {
+          auto x = X + shape;
+          auto n = cross(get<DERIVATIVE>(x));
+          auto area_correction = norm(n) / norm(cross(get<DERIVATIVE>(X)));
```

Comment:
Some comments on this calculation would be good (e.g. "This scales the dA term in the boundary integral to use the shape displaced mesh coordinates. The `norm(cross(get<DERIVATIVE>(X)))` term is already applied in the `Functional` core, so it must be adjusted using the new `x` coordinate and Jacobian).